deepcell/applications/mesmer_test.py

```diff
@@ -222,6 +224,47 @@ def test_mesmer_app(self):
             self.assertEqual(x.shape[:-1], y.shape[:-1])
             self.assertEqual(y.shape[-1], 2)
 
+            # test that kwargs are passed through successfully
+            app._predict_segmentation = Mock()
+
+            default_kwargs = {
+                'maxima_threshold': 0.1,
+                'maxima_smooth': 0,
+                'interior_threshold': 0.3,
+                'interior_smooth': 2,
+                'small_objects_threshold': 15,
+                'fill_holes_threshold': 15,
+                'radius': 2
+            }
+
+            # all defaults
+            _ = app.predict(x, compartment='whole-cell')
+            args = app._predict_segmentation.call_args[1]
+            assert args['postprocess_kwargs']['whole_cell_kwargs'] == default_kwargs
```

Comment:
I don't like that if we change the defaults in the application, we have to change the test. Let's just get the defaults from it and not make this assertion.

Reply:
Should I put them somewhere else so they can be imported?

Reply:
I think you can access them just as you did before:

```python
args = app._predict_segmentation.call_args[1]
default_kwargs = args['postprocess_kwargs']['whole_cell_kwargs']
```

Then you can make sure that the rest of wholecell/nuclear kwargs are equal to the defaults, except the new keys that were overridden.